networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -5,7 +5,6 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: CNI VRF plug-in
-include::modules/technology-preview.adoc[leveloffset=+1]
-
 include::modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc[leveloffset=+1]
+
```

Comment:
nit: do we need these extra lines?

Reply:
You're right, we don't need them, removed. Thank you!